cristin-person/src/test/java/no/unit/nva/cristin/person/handler/person.feature

```diff
@@ -61,6 +69,7 @@ Feature: API tests for Cristin Person fetch
   Scenario Outline: Fetch with unsupported Accept header returns Unsupported Media Type
     * configure headers = { 'Accept': <UNACCEPTABLE_CONTENT_TYPE> }
     Given path '/person/' + validIdentifier
+    * header Authorization = 'Bearer ' + token
```

Comment:
Do we always need authentication from now on?

Reply:
Same as above + For the accesspoints which need to authenticate against cristin we must turn on authentication to get users accessRights in the handlers requestInfo